src/routes/dictionaries/+page.svelte

```diff
@@ -30,16 +30,32 @@
   let createDictionaryError: string | null = null;
   let creatingDictionary: boolean = false;
   let files: FileList;
+  let file: File;
+  let fileInput: HTMLInputElement;
+  let isSequenceAdaptation: boolean = false;
+  let sequenceAdaptationName: string;
 
   $: hasCreatePermission = featurePermissions.commandDictionary.canCreate(data.user);
-  $: createButtonDisabled = !files;
+  $: createButtonDisabled = !files || (isSequenceAdaptation && sequenceAdaptationName === '');
+  $: {
+    if (files) {
```

Comment:
Recommend `files && files.length > 0` since `files` is an object regardless of number of files. If the user hits escape when selecting a file this will cause an error here since `file` will be undefined.